Tests/sample.ps1

```diff
@@ -0,0 +1,18 @@
+$ResourceGroupName = "dfc-test-template-rg"
+$TemplateFile = "$PSScriptRoot\..\ArmTemplates\keyvault.json"
+$params = @{ keyVaultName = "dfc-foo-bar-kv" }
+
+$output = Test-AzureRmResourceGroupDeployment `

```

Comment:
Should use a hashtable for readability and consistency

Reply:
Done